gazebo_plugins/src/gazebo_ros_camera.cpp

```diff
@@ -76,7 +76,11 @@ void GazeboRosCamera::OnNewFrame(const unsigned char *_image,
     unsigned int _width, unsigned int _height, unsigned int _depth,
     const std::string &_format)
 {
-  this->sensor_update_time_ = this->parentSensor_->LastUpdateTime();
+# if GAZEBO_MAJOR_VERSION >= 7
+  common::Time sensor_update_time = this->parentSensor_->LastMeasurementTime();
+# else
+  common::Time sensor_update_time = this->parentSensor_->GetLastMeasurementTime();
```

Comment:
Is there a use case when the kinetic-devel branch will be used with a version of Gazebo less than 7? Seems this compiler directive can be removed